src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/reported/ReportsController.kt

```diff
@@ -34,6 +34,7 @@ data class AgencyReportCountsDto(
   val transferReviewTotal: Long,
 )
 
+@PreAuthorize("hasRole('ADJUDICATIONS_REVIEWER')")
```

Comment:
some of these reports are not ALO specific.  such as my reports

Reply:
indeed - see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467#discussion_r1286774976